templates/events/details.html

```diff
@@ -403,6 +418,54 @@ <h3>Meld deg av {{ event.title }} </h3>
                 </div>
             </div>
 
+            {% if event.event_type is 1 and user_status.status or user_attending %}
+            <div id="attendees" class="modal fade">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                            <h3>Påmeldte til {{ event.title }} </h3>
+                            <h5><i>Siden dette er et sosialt arrangement kan du ønske å vise deg som påmeldt for andre å se.</i></h5>
```

Comment:
h5 shouldn't be (ab)used to change font size here. 
I'm not sure how this looks rendered, but I would create a new class (model-subtitle-ow? idk) instead.